packages/some-sass-language-server/package.yaml

```
@@ -0,0 +1,19 @@
+---
+name: some-sass-language-server
+description: Full support for @use and @forward, including aliases, prefixes and hiding. Rich documentation through SassDoc. Workspace-wide code navigation and refactoring.
+homepage: https://github.com/wkillerud/vscode-scss/tree/main/server
```

After a rename and refactor this URL has changed
```suggestion
homepage: https://github.com/wkillerud/some-sass/tree/main/packages/language-server
```